.circleci/config.yml

```diff
@@ -723,22 +312,29 @@ workflows:
       - lint-and-test
       - reprotest-wheels
       - reprotest-debs
-      - build-buster-securedrop-client
-      - build-buster-securedrop-proxy
-      - build-buster-securedrop-workstation-viewer
-      - build-buster-securedrop-export
-      - build-buster-securedrop-log
+      - build:
+          matrix:
+            parameters:
+              package:
+                - securedrop-client
+                - securedrop-export
+                - securedrop-log
+                - securedrop-proxy
+              image:
+                - "3.7-buster"
```

Comment:
Should we just drop this (and the subsequent one now)? We're not using these packages for anything. I realize there's other cleanup to do to move the changelogs over, but I'm not sure there's any point in carrying Buster-related build logic forward at this point.

Reply:
It's mostly to keep me honest and not accidentally introduce any non-parameterized OS behavior. Once this lands, the final step is for me to figure out how to have bookworm jobs OK to fail, but not block the bullseye nightlies. I'm feeling pretty confident that we'll have that done and buster gone by the end of the week.